src/utils/saveAccount.js

```diff
@@ -2,16 +2,17 @@ export const getSavedAccount = () => {
   const savedAccounts = localStorage.getItem('accounts');
   let account;
   if (savedAccounts) {
-    account = JSON.parse(savedAccounts)[0];
+    account = JSON.parse(savedAccounts);
```

Comment:
I don't see why `[0]` was removed.

Reply:
Since we aim to have multiple accounts there, I've implement the reducer to handle multiple. So I always should receive an array, even so currently the utility returns only one item. 
There are also other problems with this utility which I found it irrelevant to this PR so I didn't touch.

Reply:
then `account` should be `accounts`

Reply:
There's a ticket to fix many other issues about this utility.